src/core/TrustManager.sol

```diff
@@ -119,7 +119,17 @@ abstract contract TrustManager is IRegistry {
         // use this condition to save gas
         else if (threshold == 1) {
             AttestationRecord storage $attestation = $getAttestation({ module: module, attester: attester });
-            $attestation.enforceValid(moduleType);
+            if ($attestation.checkValid(moduleType)) return;
```

Comment:
added this routine, to handle the case that threshold == 1, but the first attester doesnt have an attestation